src/hooks/useDebounce.ts

```diff
@@ -0,0 +1,16 @@
+import {useEffect, useRef, useState} from 'react';
+
+export const useDebounce = (value: any, delay = 500) => {
+  const [debouncedValue, setDebouncedValue] = useState(value);
+  const timerRef: any = useRef();
```

Comment:
`any` -> timer type.

Reply:
Wrong file version from this PR. Deleted file.